scripts/download_elevation_grid.py

```
@@ -0,0 +1,484 @@
+                grid_points.append((lat, lon))
+                
+        return grid_points
+    
+    def estimate_elevation(self, lat: float, lon: float) -> float:
+        """Estimate elevation based on known topography."""
+        country = self.country_code
+        
+        # Country-specific elevation patterns
+        if country == 'JP':
+            return self._estimate_japan(lat, lon)
+        elif country == 'KR':
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)